tests/data/route.json

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
This is just something I sketched and saved really quickly. As we get more tests, we can either have a purpose-built input for each test, or maybe more realistically, make one or two inputs that cover a larger range of test cases.